app/src/main/java/org/mozilla/rocket/awesomebar/SearchSuggestionProvider.kt

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.rocket.awesomebar
+
+import android.os.AsyncTask
+import android.util.Log
+import mozilla.components.concept.awesomebar.AwesomeBar
+import mozilla.components.concept.fetch.MutableHeaders
+import mozilla.components.concept.fetch.Request
+import mozilla.components.concept.fetch.interceptor.withInterceptors
+import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient
+import org.json.JSONArray
+import org.mozilla.focus.BuildConfig
+import org.mozilla.focus.search.SearchEngine
+import org.mozilla.focus.utils.SupportUtils
+import org.mozilla.rocket.msrp.data.LoggingInterceptor
+import java.io.IOException
+import java.util.UUID
+
+class SearchSuggestionProvider(
+    private val searchEngine: SearchEngine,
+    private val userAgent: String,
+    private val searchUseCase: (text: String) -> Unit
+) : AwesomeBar.SuggestionProvider {
+
+    private var queryTask: AsyncTask<*, *, *>? = null
```

Comment:
what's this for?

Reply:
I'll delete this provider since we are not using it in Awesomebar